Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM centos:7
```

Comment:
Could you please add license header here and in other files in this patch too ;-) ?

Reply:
added.